config/initializers/constants.rb

```diff
@@ -1,3 +1,20 @@
 LANGUAGES = (ActiveRecord::Base.connection.table_exists? 'languages') ? Language.sorted_by_abbreviation : []
 MANY_LANGUAGES = LANGUAGES.length > 1
 TABLE_FILTER_MIN_ROWS = 10
+
+module DMPRoadmap
```

Comment:
I hear what you're saying. Not sure its particularly dangerous though. Someone would have to specifically add a Rails config line here to override anything rails is doing (e.g. `config.encoding`). 
We're using a mix of this file and values in branding.yml at the moment. We should probably have everything in one place. My vote would be to have them all in branding.yml since customizing/replacing yaml files when you deploy an instance of a Rails app is a common convention

Reply:
@briri there is potential for user confusion here because one will be able to override anything in `application.rb` from `constants.rb`. Also we can't have things like the LANGUAGES= statement (which is Rails) in `branding.yml`

Reply:
moved them back to application.rb for the time being. Moving them was not part of this bug fix I did it based on a request from @jollopre on a different issue. We can reevaluate our handling of constants post MVP